### DISCUSSION TITLE
Styling parent div element when using EditorProvider

### DISCUSSION THREAD

**Discussion by srenevey** [ueberdosis/tiptap#5656] [General Questions] [answered]
Hello,

When using the EditorProvider method to create the editor, how can I style the parent div element that's automatically added? I'm mostly interested in setting its height. 

<img width="923" alt="tiptap_div_el" src="https://github.com/user-attachments/assets/ca12ded3-a5dd-42a8-8f88-3a990d72f1f1">

When using the EditorContent approach, I can do `<EditorContent className='h-[calc(100%-3.5rem)] overflow-auto' editor={editor} />` but it seems that EditorProvider doesn't support className.

Any help would be appreciated.
Thanks,

**Comment by nperez0111** [comment, marked as answer]
The `EditorProvider` does not support this atm. I would accept a PR for this. I'd probably call it `editorContainerProps: HTMLDivElementAttributes` or whatever that type is

**Reply by kart-c** [reply]
Hey @nperez0111, let me know if I can open a PR for the same. I would be happy to do my first open source contribution.

**Reply by nperez0111** [reply]
Feel free @kart-c!

**Reply by srenevey** [reply]
Thank you

**Reply by kart-c** [reply]
Happy to help.

**Reply by nperez0111** [reply]
This is released with v2.7.4

**Comment by Equinox21040** [comment]
You could use :has css pseudo class to select the parent div 
`div:has(> .tiptap) {
    height : calc(100% - 3.5em);
}`
